src/NodaTime.Demo/OffsetDemo.cs

```diff
@@ -62,6 +62,58 @@ public void ConstructionFromTimeSpan()
             Assert.AreEqual(5400, offset.Seconds);
         }
 
+        [Test]
+        public void Add()
+        {
+            var lefHandOffset = Offset.FromHours(5);
```

Comment:
I suspect just calling these `offset1` and `offset2` is good enough here.
But if you want to keep the names, we need a "t" in left :)